docs/src/data/mockServices.tsx

```diff
@@ -0,0 +1,89 @@
+import { Amplify} from 'aws-amplify';
+
+const awsExports = {
+  aws_user_pools_id: 'xx-xxxx-x_xxxxx', // (optional) -  Amazon Cognito User Pool ID
+  aws_user_pools_web_client_id: 'xxxxxxxxxxxxxx', // (optional) - Amazon Cognito App Client ID (App client secret needs to be disabled)
+};
+Amplify.configure(awsExports)
```

Comment:
This file is really doing two things -- configure Amplify with fake data and declare mock services. Wondering if we should split this file into two.

Reply:
I would say we should move the config call to `_app.tsx` because that's the NextJS way for global configuration

Reply:
Done!